client/sites/settings/controller.tsx

```diff
@@ -18,11 +19,9 @@ export function SettingsSidebar() {
 	return (
 		<Sidebar>
 			<SidebarItem href={ `/sites/settings/site/${ slug }` }>{ __( 'Site' ) }</SidebarItem>
-			{ ! isWpcomStaging && (
-				<SidebarItem href={ `/sites/settings/administration/${ slug }` }>
-					{ __( 'Administration' ) }
-				</SidebarItem>
-			) }
```

Comment:
I don't think this change makes sense, sidebar still needs to control whether to show the item or not, moving the render decision into the sidebar item doesn't really add anything.

Reply:
Right, this was when I did the redirection inside the SidebarItem (see previous commit), will revert.

Reply:
This is now needed for this new approach.

Reply:
Why is it needed?

Reply:
We need to show it if a staging site actually lands on this path (isActive). This check happens inside SidebarItem itself.

Reply:
Ah so it ignores enabled=false if it's the active route. Ok.